perun-core/src/main/java/cz/metacentrum/perun/core/api/exceptions/InvalidHtmlInputException.java

```diff
@@ -0,0 +1,7 @@
+package cz.metacentrum.perun.core.api.exceptions;
+
+public class InvalidHtmlInputException extends PerunException{
```

Comment:
this exception could extend `WrongAttributeValueException` because the meaning of the exception is identical - the attribute value is not allowed. then we wouldn't need to add separately InvalidHtmlInputException to thrown method's exceptions.

Reply:
Sounds like a good idea. 👍